src/browser/api_protocol/api_handlers/mesh_middleware.ts

```diff
@@ -128,6 +134,84 @@ function sendMessageMiddleware(msg: MessagePackage, next: () => void) {
     }
 }
 
+const handleExternalWindow = async (identity: Identity, toGroup: Identity) => {
```

Comment:
Should type out return, like `: Promise<HWND|Error>`...

Reply:
:+1: